server/src/cloudify/parsing.ts

```diff
@@ -40,6 +41,10 @@ export function readFile (uri:string) {
     } else if (uri.startsWith('file:/')) {
         uri = uri.replace('file:/', '');
     }
+
+    uri = path.resolve(uri);
```

Comment:
Maybe we should move this line 45 (not 46) to be first step in the function (between 38 and 39)